Updated billing page to include Data Streams Monitoring (#18552)
DataDog/documentation: content/en/account_management/billing/apm_tracing_profiler.md

| [APM Host][4]      | $31 per underlying [APM host][4] per month | 1 million Indexed Spans and 150 GB of Ingested Spans included per month with every APM host.   | Datadog records the number of [APM hosts][5] you are concurrently monitoring in the Datadog APM service once an hour. On a high watermark plan (HWMP), these hourly measurements are ordered from highest to lowest at the end of the month, and Datadog charges based on the eighth highest measurement. [More information.][5] |
| APM Pro (APM Host with Data Streams Monitoring) | $35 per underlying [APM host][4]. Includes the Data Streams Monitoring. | Same as APM Host | Datadog records the number of unique APM hosts in the Datadog APM service and unique DSM hosts you are concurrently monitoring once an hour. The hourly measurements and billing for APM Pro are conducted the same as for APM Hosts.  |

> | APM Enterprise (APM Host with Data Streams Monitoring & [Continuous Profiler)][4] | $40 per underlying [APM host][4]. Includes Data Streams Monitoring, and [Continuous Profiler][6] with 4 profiled containers per host per month. | Same as APM Host | Datadog records the number of unique APM hosts in the APM service, unique DSM hosts, and unique Continuous Profiler hosts you are concurrently monitoring once per hour. The hourly measurements and billing for APM Enterprise are conducted the same as for APM Hosts. |

Reviewer: ```suggestion
| APM Enterprise (APM Host with Data Streams Monitoring & [Continuous Profiler)][6] | $40 per underlying [APM host][4]. Includes Data Streams Monitoring, and [Continuous Profiler][6] with 4 profiled containers per host per month. | Same as APM Host | Datadog records the number of unique APM hosts in the APM service, unique DSM hosts, and unique Continuous Profiler hosts you are concurrently monitoring once per hour. The hourly measurements and billing for APM Enterprise are conducted the same as for APM Hosts. |
```

| [Indexed span][5] | $1.70 per million Indexed Spans per month | Billed when usage is in excess of Indexed Spans included with every APM host | An Indexed span is an individual request against an individual service in your stack. Datadog charges based on the total number of spans indexed with retention filters or legacy Analyzed Spans to the Datadog APM service at the end of the month. [More information.][5]                                                                                          |
| [Ingested span][5] | $.10 per GB Ingested Spans per month | Billed when usage is in excess of Ingested Spans included with every APM host | An Ingested span is an individual request against an individual service in your stack. Datadog charges based on the total number of gigabytes of spans ingested to Datadog at the end of the month. [More information.][5]                                                                                          |

> **Note**: If you're using a container based environment, you get billed for the underlying host deploying the Datadog Agent.
> 
> **Note**: One profiled container is a container that is running the Continuous Profiler service. This does not include containers that are not being profiled. For instance, a DNS service container that is NOT profiled, running concurrently with your application container that IS profiled, is not counted towards the four profiler containers allotment.
> 
> **Note**: [Universal Service Monitoring][15] is included in all APM tiers (APM, APM Pro, APM Enterprise) at no additional cost.

Reviewer: ```suggestion
**Notes**:  
   - If you're using a container based environment, you get billed for the underlying host deploying the Datadog Agent.
   - One profiled container is a container that is running the Continuous Profiler service. This does not include containers that are not being profiled. For instance, a DNS service container that is NOT profiled, running concurrently with your application container that IS profiled, is not counted towards the four profiler containers allotment.
   - [Universal Service Monitoring][15] is included in all APM tiers (APM, APM Pro, APM Enterprise) at no additional cost.
```

Reviewer: Formatting change to accommodate this additional note -- worth looking into consolidating this into a separate paragraph after merging

**Sample cases illustrate annual billing rates with default 15 days Indexed Span retention. Contact [Sales][8] or your [Customer Success][9] Manager to discuss volume discounts for your account.**

### APM Hosts, indexed spans, and extra ingested spans

> Using 5 APM hosts and sending 30 million Indexed Spans, with 900 GB of total Ingested Spans.

Reviewer: ```suggestion
Using five APM hosts and sending 30 million Indexed Spans, with 900 GB of total Ingested Spans.
```

| Ingested Spans | 900 GB          | 750 GB included with 5 APM hosts. $.10 per GB for additional 150 GB of Ingested Spans.                                                                                 | 150 * $.10      | $15                  |
| Total          |            |                                                                                                 | $155 + $42.50 + $15 | **$212.50 per month** |

### APM Pro Hosts, indexed spans, and extra ingested spans 

> Using 5 APM Pro hosts and sending 30 million Indexed Spans, with 900 GB of total Ingested Spans.

Reviewer: ```suggestion
Using five APM Pro hosts and sending 30 million Indexed Spans, with 900 GB of total Ingested Spans.
```

| Billable Unit  | Quantity   | Price                                                                                           | Formula       | Subtotal              |
|----------------|------------|-------------------------------------------------------------------------------------------------|---------------|-----------------------|
| APM Pro Hosts      | 5          | $35 per host                                                                                    | 5 * $35       | $175                  |
| Indexed Spans | 30 million | 5 million included with 5 APM hosts. $1.70 per million for additional 25 million Indexed Spans | 25 * $1.70    | $42.50                |
| Ingested Spans | 900 GB          | 750 GB included with 5 APM hosts. $.10 per GB for additional 150 GB of Ingested Spans.                                                                                 | 150 * $.10      | $15                  |
| Total          |            |                                                                                                 | $175 + $42.50 + $15 | **$232.50 per month** |

### APM Enterprise Hosts with six profiled containers per host

> Using 5 APM Enterprise hosts, with the five hosts with six apps running in separate containers per each host.

Reviewer: ```suggestion
Using five APM Enterprise hosts with six apps running in separate containers per each host.
```

| Billable Unit  | Quantity   | Price                                                                                           | Formula       | Subtotal              |
|----------------|------------|-------------------------------------------------------------------------------------------------|---------------|-----------------------|
| APM Enterprise Hosts       | 5          | $40 per host                                                                                    | 5 * $40       | $200                  |
| Profiled containers  | 6 per host | $2 per additional container per host. In this case there are 6 - 4 = 2 additional containers for each host        | 2  * $2 * 5 hosts         | $20                   |
| Total          |            |                                                                                                 | $200 + $20      | **$220 per month**    |

### APM Hosts, Fargate, and indexed spans

> Using 5 APM hosts, sending 20 million Indexed Spans, and have deployed APM on average 20 Fargate Tasks over the month.

Reviewer: ```suggestion
Using five APM hosts, sending 20 million Indexed Spans, and have deployed APM on average 20 Fargate Tasks over the month.
```

Reviewer: Not sure if it's just me, but I found this sentence slightly confusing. As an optional suggestion, `and have deployed APM on an average of 20 Fargate Tasks over the month` might be a little easier to understand if that still retains the same intended meaning.

| Indexed Spans | 20 million | 5 million included with 5 APM hosts. 1.3 million included with 20 Fargate tasks. $1.70 per million for additional 13.7 million Indexed Spans | 13.7 * $1.70          | $23.29                |
| Total          |            |                                                                                                 | $155 + $40 + $23.29 | **$218.29 per month** |

### APM Enterprise Hosts, services, containers, and indexed spans

> APM Enterprise for service 1 running on container 1, service 2 running on container 2. Both Containers are running on 1 host and are sending 20 million Indexed Spans on App Analytics. 

Reviewer: ```suggestion
APM Enterprise for service 1 running on container 1, service 2 running on container 2. Both Containers are running on one host and are sending 20 million Indexed Spans on App Analytics. 
```

**Note**: The container count does not matter if the deployed Agent is on the EC2 instances.

### APM Enterprise Hosts with Kubernetes nodes and indexed spans

> APM Enterprise for apps with a Datadog Agent running on 20 worker nodes in Kubernetes sending 20 million Indexed Spans. 10 of these worker nodes have 8 pods each with 1 container per pod, the other 10 have 2 pods each with 1 container per pod. 

Reviewer: ```suggestion
APM Enterprise for apps with a Datadog Agent running on 20 worker nodes in Kubernetes sending 20 million Indexed Spans. 10 of these worker nodes have eight pods each with one container per pod, the other 10 have two pods each with one container per pod. 
```

Yes. Let Datadog know if you are interested in buying the Continuous Profiler without APM. Reach out to [Sales][8] or your [Customer Success Manager][9].

> **10. Is the Data Streams Monitoring available as a standalone product?**

Reviewer: ```suggestion
**10. Is Data Streams Monitoring available as a standalone product?**
```

> Yes. Let Datadog know if you are interested in buying Data Streams Monitoring without APM. Reach out to [Sales][8] or your [Customer Success Manager][9].

Reviewer: ```suggestion
Yes. Let Datadog know if you are interested in using Data Streams Monitoring without APM. Reach out to [Sales][8] or your [Customer Success Manager][9].
```
